FogCreek/Glitch-Community
# Profuse tea: Convert some anchor tags to Link components

## Links
https://profuse-tea.glitch.me/

## Changes:
- Convert a bunch of `<a href>` tags to `<Link to>`
- Fix a bug on /about/careers where lever only loaded the first time

## How To Test:
- Try clicking around the about pages, make sure they all load in app
- Go to the careers page, then to another page and back, make sure lever still loads
